### PR TITLE
CONSOLE-3081: updated skeleton color definitions, updated sidebar shadow

### PR DESCRIPTION
**Pull request by mattnolting**
This PR moves the skeleton color definitions to the theme file and adds a conditional wrapper to the sidebar box shadow, resets the `box shadow` to `none` only on light mode.

Before:

Screen Shot 2022-04-27 at 10 01 39 AM

After:

Screen Shot 2022-04-27 at 10 02 05 AM

Drop shadow before:

Screen Shot 2022-04-27 at 10 02 31 AM

Drop shadow after:
Screen Shot 2022-04-27 at 10 02 39 AM

### CODE REVIEW

**Comment by jcaianirh** [comment]
/label docs-approved
/label px-approved

We have docs- and px-approval for the epic.

**Review by sg00dwin — APPROVED** [review]
/lgtm

**Comment by sg00dwin** [comment]
QE Approver:
/assign @yanpzhan

**Review by rhamilto — APPROVED** [review]
/lgtm

**Pull request merged**
